app/components/auth-forms/index.jsx

```diff
@@ -0,0 +1,262 @@
+      window.socket.on('disconnect', onDisconnect)
+      window.socket.close()
+    } else cb()
+  }
+  const { classes } = props
+
+  return (
+    <div className={props.classes.authFormWrapper}>
+      <Tabs classes={classes} onLogin={onLogin} handleTabSwitch={handleTabSwitch} />
+      <form>
+        {onLogin ? (
+          <LoginForm
+            handleLogin={handleLogin}
+            hasAgreed={hasAgreed}
+            setHasAgreed={setHasAgreed}
+            setInfoMessage={setInfoMessage}
+            setFormValidationErrors={setFormValidationErrors}
+            formValidationErrors={formValidationErrors}
+            handleOnBlur={handleOnBlur}
+            handleChange={handleChange}
+            formValues={formValues}
+            infoMessage={infoMessage}
+            isDisabled={isDisabled}
+            classes={classes}
+            validationMessages={validationMessages}
+            loginErrors={loginErrors}
+          />
+        ) : (
+          <JoinForm
+            handleSignUp={handleSignUp}
+            hasAgreed={hasAgreed}
+            setHasAgreed={setHasAgreed}
+            formValidationErrors={formValidationErrors}
+            handleOnBlur={handleOnBlur}
+            handleChange={handleChange}
+            formValues={formValues}
+            infoMessage={infoMessage}
+            isDisabled={isDisabled}
+            classes={classes}
+            validationMessages={validationMessages}
+          />
+        )}
+      </form>
+    </div>
+  )
+}
+const styles = {
+  authFormWrapper: {
+    border: '0.5px solid black',
+    padding: '3rem',
+    display: 'flex',
+    flexDirection: 'column',
+    width: '30rem',
+    maxWidth: '30rem',
+    minHeight: '42rem',
+    '& form': {},
+    '& label': {
+      display: 'flex',
+      flexDirection: 'column',
+      marginTop: '3rem',
+    },
+  },
+  tabs: {
+    display: ' flex',
+    width: '100%',
+    pointerEvents: 'auto',
+    cursor: 'pointer',
+    color: '#18397D',
+    fontWeight: '900',
+    fontSize: '2rem',
+  },
+  activeTab: {
+    backgroundColor: 'white',
+    width: '50%',
+  },
+  nonActiveTab: {
+    backgroundColor: '#E5E5E5',
+    width: '50%',
+  },
+  hide: {
+    display: 'none',
+  },
+  disable: {
+    backgroundColor: '#D3D3D3',
+    float: 'right',
+    width: '9rem',
+    height: '3rem',
+    fontSize: '1.5rem',
+    fontWeight: '600',
+  },
+  activeBtn: {
+    backgroundColor: '#E5A650',
+    cursor: 'pointer',
+    float: 'right',
+    width: '9rem',
+    height: '3rem',
+    color: 'white',
+    fontSize: '1.5rem',
+    fontWeight: '600',
+  },
+  agreementWrapper: {
+    display: 'flex',
+    alignItems: 'center',
+    cursor: 'pointer',
+    color: '#18397D',
+    marginTop: '1rem',
+    '& span': {
+      marginLeft: '0.5rem',
+      fontWeight: '800',
+    },
+  },
+  formValidationErrors: {
+    color: 'red',
+    width: '100%',
+  },
+}
+export const AuthForm = injectSheet(styles)(Temp)
```

Comment:
this is causing injectSheet to be called twice, and creating 2 object, AuthForm and default.
I think you can do export default AuthForm and get the desired result.  You may also be able to say export default const AuthForm = ... but maybe that only works with classes.